services/api/src/models/shop.json

```diff
@@ -0,0 +1,15 @@
+{
+  "attributes": {
+    "name": { "type": "String", "trim": true, "required": true },
+    "description": { "type": "String", "trim": true },
+    "images": [
+      {
+        "type": "ObjectId",
+        "ref": "Upload",
+        "autopopulate": true
+      }
+    ],
+    "categories": [{ "type": "ObjectId", "ref": "Category", "autopopulate": true }],
```

Comment:
I don't think prettier overrides JSON formatting does it?
Would be nice if this was more readable like normal JSON

Reply:
it does

Reply:
looks like it's a vscode thing... if you run it on command line it doesn't change it...
vscode embedded prettier is wonky man I've had other issues with it

Reply:
Prettier seems to only kick in for .js files and not .json files in my VSCode. Would like it if we could fix that

Reply:
as long as it doesn't fold lines